packages/react-tinacms-editor/src/node-views/ImageView/index.ts

```diff
@@ -35,7 +42,7 @@ export class ImageView implements NodeView {
     this.dom.classList.add('tinacms-image-wrapper')
     this.img = document.createElement('img')
     const { src, align, alt, title, width, height } = node.attrs
-    this.img.src = src
+    this.img.src = previewUrl(src)
```

Comment:
NodeView will use `previewUrl` to get url for image. Default will be identity function.